.changeset/warm-planes-decide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/commercial': minor
```

Comment:
A change to peer dependencies requires a major bump

Reply:
Thanks @domlander - I have altered that.